### PR TITLE
New Endpoint for full data transfer

### PR DESCRIPTION
**Pull request by stjosh**
See https://github.com/swissfintechinnovations/ca-mortgage/issues/49

### CODE REVIEW

**Review by rolfwagner — CHANGES_REQUESTED** [review]
Looks ok, at first glance.
Please rename  /cases to /mortage-cases*

**Comment by stjosh** [comment]
@rolfwagner Thanks for your feedback - I have renamed the enpoints.

**Comment by rolfwagner** [comment]
@olegkovalov Ok for MP? Scope rls 3. We will discuss it at the next mortgage meeting.

**Review by olegkovalov — APPROVED** [review]

File: mortgageAPI.yaml [hunk 1]
```diff
@@ -37,6 +39,104 @@ security:
       - read
       - write
 paths:
+  /mortgage-cases:
+    get:
+      tags:
+        - Full Mortgage Cases
+      description: List of mortgage cases for the authenticated contiext including the corresponding properties, sorted by caseId
```
[the remark this answers is not in the document]
Comment (stjosh) [reply mortgageAPI.yaml]: Many thanks, good catch. I have just fixed it.

**Comment by stjosh** [comment]
@rolfwagner So I guess we're ready to merge? 😃

**Review by rolfwagner — APPROVED** [review]
ok

**Pull request merged**